src/Orleans/Runtime/GrainReference.cs

```diff
@@ -320,7 +324,26 @@ private Task<object> InvokeMethod_Impl(InvokeMethodRequest request, string debug
         {
             if (debugContext == null && USE_DEBUG_CONTEXT)
             {
-                debugContext = string.Intern(GetDebugContext(this.InterfaceName, GetMethodName(this.InterfaceId, request.MethodId), request.Arguments));
+                if (USE_DEBUG_CONTEXT_PARAMS)
```

Comment:
Right. But why do u need a special handling for it? Just do the same as for regular debug context.